erc/ERCS/erc-7562.md

```diff
@@ -317,19 +317,24 @@ These are storage rules that help to protect the bundler from denial of service:
   This means that `Paymaster` and `Factory` contracts cannot practically be an "account" contract as well.
 * **[STO-041]** `UserOperation` may not use associated storage (of either its account or from staked entity) in a contract that is a "sender" of another UserOperation in the mempool.
 
-### Staked Entities Reputation Rules
 
-* **[SREP-010]** The "canonical mempool" defines a staked entity if it has `MIN_STAKE_VALUE` and unstake delay of `MIN_UNSTAKE_DELAY`
-* **[SREP-020]** A `BANNED` address is not allowed into the mempool.\
+### General Reputation rules
+The following reputation rules apply for both staked and unstaked entities.
+
+* **[GREP-010]** A `BANNED` address is not allowed into the mempool.\
   Also, all existing `UserOperations` referencing this address are removed from the mempool.
-* **[SREP-030]** A `THROTTLED` address is limited to:
+* **[GREP-020]** A `THROTTLED` address is limited to:
     * `THROTTLED_ENTITY_MEMPOOL_COUNT` entries in the mempool.
     * `THROTTLED_ENTITY_BUNDLE_COUNT` `UserOperations` in a bundle.
     * Can remain in the mempool only for `THROTTLED_ENTITY_LIVE_BLOCKS`.
-* **[SREP-040]** An `OK` staked entity is unlimited by the reputation rule.
+* **[GREP-040]** If an entity fails the second validation or fails bundle creation, its `opsSeen` set to `10000`, and `opsIncluded` to zero, causing it to be `BANNED`.
```

Comment:
User constant instead of 10000

Reply:
Maybe this number should be derived from the other constants?

Reply:
What's GREP?
and we are **still** missing a calculation for the correct opsSeen and opsIncluded for a specific duration of banning.  We only know that the current one is wrong.

Suppose we want to ban for 24 hours, we need to calculate opsSeen and opsIncluded such that after 23 hourly decalys it'll still be in BANNED category, but after 24 hourly decays it'll enter THROTTLED category.

Reply:
and why was SREP-040 removed?  Is it no longer the case that we don't limit staked entities while they are OK?

Reply:
It was not removed, just renumbered